lib/doActionAwaitingNavigation.js

```diff
@@ -112,10 +112,10 @@ const waitForNavigation = (timeout, promises = []) => {
             );
           },
           defaultConfig.retryInterval,
-          defaultConfig.retryTimeout,
+          timeout,
         )
           .then(resolve)
-          .catch(reject);
+          .catch(() => reject('Timedout'));
```

Comment:
Won't this cause all errors to be rejected with `Timedout`?

Reply:
only waiting for readyState to be complete will get rejected with Timedout